src/UI/Components/ServiceInstanceForm/ServiceInstanceForm.test.tsx

```diff
@@ -24,6 +24,9 @@ const setup = (
     | EnumField
   )[],
   func: undefined | jest.Mock = undefined,
+  isEdit = false,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  originalAttributes: any = undefined,
```

Comment:
I know that's not much better than any but if I recall correctly for attributes we've got `InstanceAttributeModel` which is still a little enigmatic but we don't need to overwrite eslint

Reply:
updated